### PR TITLE
DP-32334: add teams alert when Circuit breaker is enabled.

### PR DESCRIPTION
**Pull request by justinLCM**
Add Teams alerts for when an ECS service enables Circuit Breaker.

### CODE REVIEW

File: ecs_fargate/ecs.tf [hunk 1]
```diff
@@ -278,4 +279,23 @@ resource "aws_sns_topic_subscription" "cb_email_targets" {
...
+  topic_map = [
+    {
+      topic_arn  = aws_sns_topic.cb.arn
+      human_name = join("", [var.ecs_cluster_name, var.ecs_service_name, "Teams", "Alert"])
+      icon_url   = "https://static.vecteezy.com/system/resources/previews/024/392/058/non_2x/alert-mark-failed-to-load-something-went-wrong-tap-to-retry-concept-illustration-flat-design-eps10-simple-graphic-element-for-landing-page-empty-state-ui-infographic-icon-vector.jpg"
+    }
+  ]
+  tags = {
+    "application" = "massgov-teams-notifications"
+  }
```
Comment (jaredhm) [inline ecs_fargate/ecs.tf]: pass `var.tags` thru

File: ecs_fargate/ecs.tf [hunk 2]
```diff
@@ -278,4 +279,23 @@ resource "aws_sns_topic_subscription" "cb_email_targets" {
   topic_arn = aws_sns_topic.cb[0].arn
   protocol  = "email"
   endpoint  = var.ecs_circuit_breaker_alert_email[count.index]
+}
+
+
+module "massgov_teamsalerts" {
```
Comment (jaredhm) [inline ecs_fargate/ecs.tf]: small thing, but this isn't necessarily for massgov

**Review by jaredhm — APPROVED** [review]

File: ecs_fargate/ecs.tf (hunk 2 above)
Comment (justinLCM) [reply ecs_fargate/ecs.tf]: renamed

File: ecs_fargate/ecs.tf (hunk 1 above)
Comment (justinLCM) [reply ecs_fargate/ecs.tf]: removed

**Pull request merged**